web/src/components/InformationCards/ItemInformationCard/FieldsDisplay.tsx

```
@@ -31,29 +48,29 @@ const FieldsDisplay: React.FC<IFieldsDisplay> = ({ props }) => {
       {imageFields.length || textFields.length ? (
         <FieldsContainer>
           {imageFields.map((field) => (
-            <ItemField {...field} detailed />
+            <ItemField {...field} detailed isUnrecognized={isUnrecognized(field)} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)